scripts/start.js

```diff
@@ -17,6 +14,11 @@ process.on('unhandledRejection', err => {
 // Ensure environment variables are read.
 require('../config/env');
 
+process.env.DEV_GANACHE_HOST = (process.env.DEV_GANACHE_HOST === undefined) ? 'localhost' : process.env.DEV_GANACHE_HOST;
```

Comment:
@lalexgap do you think we need the explicit `=== undefined` here? If `DEV_GANACHE_HOST` is null or an empty string, do we still want to set it to the default `localhost`? If so, we could just use `process.env.DEV_GANACHE_HOST = (process.env.DEV_GANACHE_HOST) ? 'localhost' : process.env.DEV_GANACHE_HOST;`

Reply:
javascript doesn't like `process.env.DEV_GANACHE_HOST = (process.env.DEV_GANACHE_HOST) ? 'localhost' : process.env.DEV_GANACHE_HOST;` but I've changed it to be cleaner and use the `||` syntax which should catch empty/null values as well